app/routes/events/components/EventAdministrate/Abacard.js

```diff
@@ -25,26 +26,33 @@ class Abacard extends React.Component<*, State> {
   };
 
   handleSelect = (result: SearchResult) => {
-    this.props.handleSelect(result).then(
-      () => {
-        const sound = new window.Audio(goodSound);
-        sound.play();
-        this.showCompleted();
-        return this.props.clearSearch();
-      },
-      err => {
-        const payload = _.get(err, 'payload.response.jsonData');
-        if (payload && payload.errorCode === 'not_registered') {
-          alert('Bruker er ikke påmeldt på eventet!');
-        } else if (payload && payload.errorCode === 'already_present') {
-          alert('Bruker er allerede satt som tilstede.');
-        } else {
-          alert(
-            `Det oppsto en uventet feil: ${JSON.stringify(payload || err)}`
-          );
+    this.props.clearSearch();
+    this.props
```

Comment:
Is there a reason to not return this promise?

Reply:
Nah, not really - fixed